fastlane_bot/config/constants.py

```diff
@@ -8,5 +8,7 @@
 FLASHLOAN_FEE_MAP = {
     "ethereum": 0,
     "coinbase_base": 0,
-    "fantom": 0.0003
+    "fantom": 0.0003,
+    "mantle": 0,
 }
+MANTLE_NAME = "mantle"
```

Comment:
Also declared in each one of the following places:
- MANTLE = "mantle" (global in file run_blockchain_terraformer.py)
- NETWORK_NAME = "mantle" (in class _ConfigNetworkMantle in file network.py)
- NETWORK_MANTLE = "mantle" (global in file selectors.py)

While this one here is probably the least suitable.
Can you please remove it from this file, and then where importing it is required - import it from some other file?